<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>HyperLink Tool</string>
     </attribute>
     <widget class="QPushButton" name="importPrimaryButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Import Primary Video</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Actions: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="importSecondaryButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Import Secondary Video</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createNewHyperLink">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Create New Hyperlink</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Link:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selectLinks">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="connectVideo">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect Video</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveFile">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>50</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save File</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderLeft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>721</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="frameLabelLeft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PrimaryFrame:</string>
      </property>
     </widget>
     <widget class="QLabel" name="frameLabelRight">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Secondary Frame: </string>
      </property>
     </widget>
     <widget class="QLabel" name="frameCountLeft">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>460</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="frameCountRight">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>540</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="primaryVideoProgressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>721</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="secondaryVideoProgressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>721</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderRight">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>721</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Interactive Video Player</string>
     </attribute>
     <widget class="QPushButton" name="playerPlay">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playerPause">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>180</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playerStop">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>230</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Playing Frame</string>
      </property>
     </widget>
     <widget class="QLabel" name="frameCountPlayer">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>440</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QSlider" name="sliderPlayer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="playerVideoProgressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
